src/components/modalWindowContent/modalWindowContent.js

```diff
@@ -0,0 +1,281 @@
+import './modalWindowContent.css';
+
+/* EXAMPLE DATA 
+
+const answersList = [
+  {
+    id: 1,
+    img: '../../../static/assets/img/modes/people/13.jpg',
+    correctAnswer: 'Chewbacca',
+    human: {
+      answer: 'Chewbacca',
+      isCorrect: true
+    },
+    autoPlayer: {
+      answer: 'Darth Vader',
+      isCorrect: false
+    }
+  },
+  {
+    id: 2,
+    img: '../../../static/assets/img/modes/people/7.jpg',
+    correctAnswer: 'Beru Whitesun lars',
+    human: {
+      answer: 'Beru Whitesun lars',
+      isCorrect: true
+    },
+    autoPlayer: {
+      answer: 'Darth Vader',
+      isCorrect: false
+    }
+  },
+  {
+    id: 3,
+    img: '../../../static/assets/img/modes/people/19.jpg',
+    correctAnswer: 'Jek Tono Porkins',
+    human: {
+      answer: 'Chewbacca',
+      isCorrect: false
+    },
+    autoPlayer: {
+      answer: 'Jek Tono Porkins',
+      isCorrect: true
+    }
+  },
+  {
+    id: 4,
+    img: '../../../static/assets/img/modes/people/26.jpg',
+    correctAnswer: 'Lobot',
+    human: {
+      answer: 'Lobot',
+      isCorrect: true
+    },
+    autoPlayer: {
+      answer: 'Darth Vader',
+      isCorrect: false
+    }
+  }
+]; */
+
+const hideModal = () => {
+  const modal = document.querySelector('.content');
+  modal.classList.add('hidden');
+};
+
+const countCorrectAnswers = (answersList, player) => {
+  const answers = [];
+  answersList.forEach((answer) => {
+    if (answer[player].isCorrect) {
+      answers.push(answer[player].isCorrect);
+    } else return;
+  });
+  return answers;
+};
+
+const getYodaQuote = (humanPlayer, autoPlayer) => {
+  if (humanPlayer > autoPlayer) {
+    return 'The force is strong in you young Padawan!';
+  } else return 'Much to learn you still have...my young Padawan.';
+};
+
+const getStringForEndGame = (
+  quote,
+  questionsNumber,
+  humanAnswers,
+  autoPlayerAnswers
+) => {
+  return `${quote} During 1 minute you have answered ${humanAnswers} / ${questionsNumber} questions. And Google guessed ${autoPlayerAnswers} / ${questionsNumber}.`;
+};
+
+const getTextForEndGame = (answersList) => {
+  const questionsNumber = answersList.length;
+
+  const human = countCorrectAnswers(answersList, 'human').length;
+  const autoPlayer = countCorrectAnswers(answersList, 'autoPlayer').length;
+
+  const text = getStringForEndGame(
+    getYodaQuote(human, autoPlayer),
+    questionsNumber,
+    human,
+    autoPlayer
+  );
+  return text;
+};
+
+const createTitle = () => {
+  const title = document.createElement('h2');
+  title.classList.add('content_title');
+  title.innerText = 'GAME OVER';
+  return title;
+};
+
+const createSummary = (answersList) => {
+  const summary = document.createElement('p');
+  summary.classList.add('content_summary');
+  summary.innerText = getTextForEndGame(answersList);
+  return summary;
+};
+
+const createHeroImg = () => {
```

Comment:
hero in sense of what is shown on image or element placement?

Reply:
hero as of what is shown on the image, I thought it can be changed for a different one, but actually, I don't know now if it makes any sense. Could be just YodaImg :P